cdap-ui/app/cdap/components/CaskWizards/OneStepDeploy/Deploy/Deploy.scss

```diff
@@ -17,4 +17,24 @@
 .deploy-step {
   height: calc(100% - 45px);
   width: 100%;
+
+  .file-drop-container {
+    pointer-events: none;
+
+    .fa-upload {
+      opacity: 0.5;
+    }
+  }
+}
+
+.cask-wizard {
+  .wizard-body {
+    .wizard-steps-content {
+      .wizard-navigation {
+        .btn {
+          margin-top: 10px;
```

Comment:
I think this has caused an un-necessary overflow scroll to appear in other wizards.